src/test/java/com/example/rets_api/integration/RetsControllerIntegrationTest.java

```diff
@@ -97,6 +101,37 @@ public void getPropertiesByParams() {
         assertEquals(propertyDTORequest, getPropertyDTOFromResponse(responseEntityGet, 0));
     }
 
+    @Test
+    public void step40deletetPropertyById() {
+        //Preparing scenario: Adding a property
+        PropertyDTO propertyDTORequest = createPropertyDTOWithBasicFields();
+        String URL = "http://localhost:" + port + "/properties";
+        ResponseEntity<Long> responseEntityPost = this.restTemplate
+                .postForEntity(URL, propertyDTORequest, Long.class);
+        assertEquals(200, responseEntityPost.getStatusCodeValue());
+
+        //Creating objects
+        String URLWithID = URL
+                .concat("/")
+                .concat(String.valueOf(responseEntityPost.getBody()));
+
+        //Request
+        ResponseEntity<PropertyDTO> responseEntityGet = this.restTemplate
```

Comment:
You don't need this request. You just need to create a property and remove the property, don't need to get the property

Reply:
just want to make sure the property is there, I added it to make sure the property has been created. if you think it is better to be deleted, please let me know

Reply:
deleted

Reply:
the return 200 and the id in the creation is enough to be sure that the property was created ;)